scripts/stratis_udev/stratis_uuids_to_names.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python3
+#
```

Comment:
I have this recollection that somebody told us that it would be just as good, probably even better, to write this in Rust and install the executable, but I can't remember quite who said that. I'm not sure how doing that would influence our decision about where to put it.

Reply:
Yes, the recommendation is that udev shell out as little as possible. I was thinking of rewriting this in Rust so does that change your thought on where we should put it in the repository?

Reply:
Right now I'm inclining toward moving the existing Python script to somewhere less canonical, since it's for internal use only. ```scripts``` is a pretty usual designation for things that might go in a release. Maybe ```developer_toolbox``` would be a better choice for a place to stow the existing script; that makes it pretty obvious that its not for regular distribution.

But that doesn't exactly solve the problem of where this should go if it were in Rust.

Reply:
A quick look says that our bin directory might not be such a bad idea.